src/routes/safes/safes.controller.ts

```diff
@@ -42,7 +42,11 @@ export class SafesController {
     return this.service.getNonces({ chainId, safeAddress });
   }
 
-  @ApiQuery({ name: 'wallet_address', required: false })
+  @ApiQuery({ name: 'wallet_address', required: false, type: String })
+  @ApiQuery({ name: 'currency', required: true, type: String })
+  @ApiQuery({ name: 'safes', required: true, type: String })
+  @ApiQuery({ name: 'trusted', required: false, type: Boolean })
+  @ApiQuery({ name: 'exclude_spam', required: false, type: Boolean })
   @Get('safes')
   async getSafeOverview(
     @Query('currency') currency: string,
```

Comment:
TODO: When parameters are optional, mark them as optional in the code by appending a question mark (?) to the end of the parameter name. This way, Swagger may automatically recognize the parameter as optional. e.g.

```suggestion
    @Query('currency') currency?: string,
```
Implementing this requires additional effort and is outside the scope of the current PR.

Reply:
Which parameters are not marked as optional? In this case, `currency` is required.

Reply:
I have seen some instances, but here for example `excludeSpam`. It has a default value though.

Reply:
Perhaps we should reconsider usage of the default pipes, and instead assign a default value manually. This would negate the requirement of `ApiQuery`. I'll merge this for now and we can reconsider this later.